### DISCUSSION TITLE
Collection of requirements, ideas, and features for Harbor 2.12

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [goharbor/harbor#20666] [Ideas]
Hello Community,

As we released v2.11 and we have plan and roadmap for v2.12 it's time to brainstorm the ideas and requirements for the feature list for 2.9!

Previous discussion can be found here: https://github.com/goharbor/harbor/discussions/19791
Please take a look for all candidates for 2.11 that will be transferred to 2.12:
[goharbor/harbor/issues (is:open sort:updated-desc label:candidate/2.11.0)](https://github.com/goharbor/harbor/issues?q=is%3Aopen+sort%3Aupdated-desc+label%3Acandidate%2F2.12.0+)
And 2.9 candidates already added:
[goharbor/harbor/issues (is:open sort:updated-desc label:candidate/2.11.0)](https://github.com/goharbor/harbor/issues?q=is%3Aopen+sort%3Aupdated-desc+label%3Acandidate%2F2.12.0+)

Please share your thoughts here so we can all work on proper roadmap and achieving it!
Keep in mind that Harbor is community developed and driven project so not everything can be implemented right away :)
If you have an idea or need and eager to work on, we will support you and help you get there!

Check the v2.12 schedule/timeline here. https://github.com/orgs/goharbor/projects/1#column-19954948

Thank you being essential part of the community!

Orlix
Harbor Community Manager

**Comment by DingGGu** [comment]
Hi there!

1. How about new feature: project policy for Any Member prevent (block) create a Public Project?

This is useful for prevent data leak under unexpected and misunderstand harbor project permission for Anyone new to harbor.

2. Any forward discussions for Support IRSA on harbor-registry? This issue was created a long time ago, and It makes hard to manage Infrastrucutre and running harbor on AWS (S3).

I know that registry depends on distribution/distribution, but that project has a very slow release cycle.

Harbor is such a popular and great project that I think it's time to make a decision to support IRSA.

**Comment by snakebyte91** [comment]
We would be very happy about the support of the GitLab registry as proxy cache repo, [#12885](https://github.com/goharbor/harbor/issues/12885)

IRSA support would also be great.

**Comment by hyeyoung-leee** [comment]
Hi,

Providing the functionality to use the AND logic in the retention policy would be very beneficial.
[#20403](https://github.com/goharbor/harbor/issues/20403)

**Comment by kingbj940429** [comment]
IRSA support would also be great.
